FHSDKPhone81/Services/Auth/OAuthClientHandlerService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Security.Authentication.Web;
+
+namespace FHSDK.Services
+{
+    /// <summary>
+    /// OAuth login handler for windows phone
+    /// </summary>
+    class OAuthClientHandlerService : OAuthClientHandlerServiceBase
+    {
+        private TaskCompletionSource<OAuthResult> tcs;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public OAuthClientHandlerService()
+            : base()
+        {
+
+        }
+
+        public override Task<OAuthResult> Login(string oauthLoginUrl)
+        {
+            tcs = new TaskCompletionSource<OAuthResult>();
+            Uri uri = new Uri(oauthLoginUrl, UriKind.Absolute);
+            var oauth = await WebAuthenticationBroker.AuthenticateAsync(WebAuthenticationOptions.None, uri);
```

Comment:
I am getting the following compiler error here: 
The 'await' operator can only be used within an async method. Consider marking this method with the 'async' modifier and changing its return type to 'Task<System.Threading.Tasks.Task<FHSDK.Services.OAuthResult>>'.	C:\Users\FeedHenry\Documents\GitHub\fh-dotnet-sdk\FHSDKPhone81\Services\Auth\OAuthClientHandlerService.cs.